defuse/src/contract/tokens/nep245/deposit.rs

```diff
@@ -115,19 +118,29 @@ impl Contract {
     #[allow(clippy::needless_pass_by_value)]
     pub fn mt_resolve_deposit(
         &mut self,
-        receiver_id: &AccountId,
-        token_ids: Vec<TokenId>,
-        deposited_amounts: Vec<U128>,
+        receiver_id: AccountId,
+        contract_id: AccountId,
+        amounts: Vec<(defuse_nep245::TokenId, U128)>,
```

Comment:
maybe use two separate params to avoid complexity with `.unzip()` indise?
```suggestion
        token_ids: Vec<defuse_nep245::TokenId>,
        amounts: Vec<U128>,
```

Reply:
it's `#[private]`, so it's like you're calling this function the same way you call `resolve_deposits_internal()` below with unzipped iter